src/components/Inventory/view/BatchesCellRenderer.tsx

```diff
@@ -41,29 +52,102 @@ export default function BatchesCellRenderer(props: RendererProps<TableRowType>):
     return <Typography className={classes.text}>{iValue}</Typography>;
   };
 
+  const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
```

Comment:
possible to rename functions like handleClick and setAnchorEl to something more specific. Basically, it will help to know that these are specific to change quantity features - since batches cell renderer applies to all the cells.